src/Sulu/Bundle/ContentBundle/Resources/views/Template/content-types/url.html.twig

```diff
@@ -1,13 +1,16 @@
 <div id="{{ id|raw }}"
      data-property='{{ property|json_encode }}'
      data-mapper-property="{{ property.name }}"
-     data-aura-component="input@husky"
-     data-aura-skin="url"
+     data-aura-component="url-input@husky"
      data-aura-instance-name="{{ id|raw }}"
-     data-aura-input-id="{{ id|raw }}"
-     data-type="husky-input"
+     data-aura-input-class="no-preview-update"
+     data-aura-scheme="{{ params.defaults.value.scheme.value | default('http://') }}"
+     data-aura-specific-part="{{ params.defaults.value.specific_part.value | default('') }}"
+     data-aura-schemes="{{ sulu_parameter_to_names_array(params.schemes.value) | json_encode }}"
+     data-type="url-input"
+     data-type-schemes="{{ sulu_parameter_to_names_array(params.schemes.value) | json_encode }}"
      data-form="true"
-     class="preview-update trigger-save-button"
+     class="preview-change-update trigger-save-button"
```

Comment:
with this class the preview hears on the dom change event to detect changed data. this event is new and introduced to decouple content type from preview. the content type only throws the dom event "change" when the data was changed. i will document that when we write the chapter "how to implement custom content types".